ci/cpu/cudf/upload-anaconda.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Adopted from https://github.com/tmcdonell/travis-scripts/blob/dfaac280ac2082cd6bcaba3217428347899f2975/update-accelerate-buildbot.sh
+
+set -e
+
+if [ "$BUILD_CUDF" == "1" ]; then
+  if [ "$BUILD_ABI" == "1" ]; then
+    export UPLOADFILE=`conda build conda/recipes/cudf -c rapidsai -c nvidia -c numba -c conda-forge -c defaults --python=$PYTHON --output`
+  else
+    export UPLOADFILE=`conda build conda/recipes/cudf -c rapidsai/label/cf201901 -c nvidia/label/cf201901 -c numba -c conda-forge/label/cf201901 -c defaults --python=$PYTHON --output`
+  fi
+
+  SOURCE_BRANCH=master
+
+  # Have to label all CUDA versions due to the compatibility to work with any CUDA
+  if [ "$LABEL_MAIN" == "1" -a "$BUILD_ABI" == "1" ]; then
+    LABEL_OPTION="--label main --label cuda9.2 --label cuda10.0"
+  elif [ "$LABEL_MAIN" == "0" -a "$BUILD_ABI" == "1" ]; then
+    LABEL_OPTION="--label dev --label cuda9.2 --label cuda10.0"
+  elif [ "$LABEL_MAIN" == "1" -a "$BUILD_ABI" == "0" ]; then
+    LABEL_OPTION="--label cf201901 --label cf201901-cuda9.2 --label cf201901-cuda10.0"
+  elif [ "$LABEL_MAIN" == "0" -a "$BUILD_ABI" == "0" ]; then
+    LABEL_OPTION="--label cf201901-dev --label cf201901-cuda9.2 --label cf201901-cuda10.0"
+  else
+    echo "Unknown label configuration LABEL_MAIN='$LABEL_MAIN' BUILD_ABI='$BUILD_ABI'"
+    exit 1
+  fi
+  echo "LABEL_OPTION=${LABEL_OPTION}"
+
+  test -e ${UPLOADFILE}
+
+  # Restrict uploads to master branch
+  if [ ${GIT_BRANCH} != ${SOURCE_BRANCH} ]; then
+    echo "Skipping upload"
+    return 0
+  fi
+
+  if [ -z "$MY_UPLOAD_KEY" ]; then
+    echo "No upload key"
+    return 0
+  fi
+
+  echo "Upload"
+  echo ${UPLOADFILE}
+  anaconda -t ${MY_UPLOAD_KEY} upload -u rapidsai ${LABEL_OPTION} --force ${UPLOADFILE}
+fi
```

Comment:
Do we want to invoke conda-build in our CI system?  I know that this was here before.  I'd just like to ask if maybe we want to remove it?

Reply:
We do use `conda build` in the other scripts to kick off the builds, but use `anaconda upload` to upload only select packages and in the pipelines of cuDF/cuML we have multiple conda builds that all need to succeed before uploading. That way we can prevent uploading partial sets of packages when there is a failure.

Reply:
My question is why we're building conda packages and uploading them in the testing CI at all

Reply:
We are building conda packages to ensure during the PR process that once merged they will build them correctly. This prevent us from merging a PR that then breaks a conda build. Also we do not upload during builds for PRs or the `branch-M.X` builds. We do reuse the same code and when it is being built from the master branch it auto uploads. The logic is contained in the rest of this file.